rfb/rfbclient.h

```diff
@@ -471,6 +471,9 @@ typedef struct _rfbClient {
 	 * For internal use only.
 	 */
 	MUTEX(tlsRwMutex);
+
+	rfbBool requestedResize;
+	rfbExtDesktopScreen screen;
```

Comment:
AFAICT only the rfbExtDesktopScreen's width and height are used. Why not simply use client->width and client->height? Because they represent the framebuffer dimensions and this one here the inteded dimensions? Probably you're more into this and can add an explaining comment?

Reply:
Exactly. `rfbExtDesktopScreen screen;` is used for intended dimension, instead `client->width` and `client->height` are used to manage the real framebuffer dimensions.

In future also more variables of rfbExtDesktopScreen could be used.